docs/upgrades/v0.14-to-v1-upgrade/examples/src/test/kotlin/examples/TypeChanges.kt

```diff
@@ -0,0 +1,104 @@
+package examples
```

Comment:
self-review: future migration guide examples will likely go in their own separate file. E.g. for AST changes, we'd create a test file called `AstChanges.kt` under the `examples` and `upgraded-examples` directories.